src/GNNGraphs/gnngraph.jl

```diff
@@ -5,11 +5,11 @@ https://juliagraphs.org/Graphs.jl/latest/types/#AbstractGraph-Type
 https://juliagraphs.org/Graphs.jl/latest/developing/#Developing-Alternate-Graph-Types
 =============================================#
 
-const COO_T = Tuple{T, T, V} where {T <: AbstractVector{<:Integer}, V}
-const ADJLIST_T = AbstractVector{T} where T <: AbstractVector{<:Integer}
+const COO_T = Tuple{T,T,V} where {T<:AbstractVector{<:Integer},V}
+const ADJLIST_T = AbstractVector{T} where {T<:AbstractVector{<:Integer}}
```

Comment:
there are a lot of unrelated code formatting changes in this PR. It is fine and you can keep them, but for the future, these changes should be done in a separate PR since they make it harder to review the actual changes in the PR.